quickget

```diff
@@ -2633,6 +2644,27 @@ function get_reactos() {
     echo "${URL} ${HASH}"
 }
 
+function get_rebornos() {
+    local ISO=""
+    local HASH=""
+    ISO=$(wget -q -O- "https://meta.cdn.soulharsh007.dev/RebornOS-ISO?format=json" | jq -r ".url")
+    HASH=$(wget -q -O- "https://meta.cdn.soulharsh007.dev/RebornOS-ISO?format=json" | jq -r ".md5")
+    echo "${ISO} ${HASH}"
+}
+
+function get_redox() {
+    local EDITION="${1:-}"
+    local HASH=""
+    local ISO=""
+    local URL=""
+    URL="https://static.redox-os.org/img/x86_64"
+    # For now number is set manually, will see how often will be updated..
+    NUMBER="2024-04-18_975"
+    ISO="redox_${EDITION}_x86_64_${NUMBER}_livedisk.iso.zst"
+    HASH=$(wget -qO- https://static.redox-os.org/img/x86_64/SHA256SUM | grep ${ISO} | cut_1)
+    echo "${URL}/${ISO} ${HASH}"
```

Comment:
```suggestion
    local HASH=""
    local ISO=""
    local URL="https://static.redox-os.org/img/x86_64"
    local DATA

    DATA="$(curl -s "${URL}/SHA256SUM" | grep ".iso" | grep "${EDITION}")"
    ISO="$(echo ${DATA}" | cut -d '*' -f2)"
    HASH="$(echo "${DATA}" | cut_1

    echo "${URL}/${ISO} ${HASH}"
```

Reply:
we need use this parametrs:
```
SDL_VIDEO_X11_DGAMOUSE=0 qemu-system-x86_64 -d cpu_reset,guest_errors -smp 4 -m 2048 \
    -chardev stdio,id=debug,signal=off,mux=on,"" -serial chardev:debug -mon chardev=debug \
    -machine q35 -device ich9-intel-hda -device hda-duplex -netdev user,id=net0 \
    -device e1000,netdev=net0 -device nec-usb-xhci,id=xhci -enable-kvm -cpu host \
    -drive file=`echo $HOME/Downloads/redox_demo_x86_64*_harddrive.img`,format=raw
```

Cn you help mod quickemu...